packages/react-components/react-dialog/src/components/Dialog/Dialog.types.ts

```diff
@@ -87,6 +87,13 @@ export type DialogProps = ComponentProps<Partial<DialogSlots>> & {
    * Alternatively can only contain {@link DialogSurface} if using trigger outside dialog, or controlling state.
    */
   children: [JSX.Element, JSX.Element] | JSX.Element;
+  /**
+   * Enables standard behavior according to the [HTML dialog spec](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal)
+   * where the focus trap involves setting outside elements inert.
```

Comment:
I'd mention the consequence of `inert` - that the user can tab into the browser window